File: javascript/easy/1-bitAnd2-bitCharacters.js
/**
 * @param {number[]} bits
 * @return {boolean}
 */
var isOneBitCharacter = function(bits) {
    let oneBit = false, i = 0;
    while(i < bits.length) {
        if(bits[i] == 1) {
            oneBit = false;
            i++;
        } else {
            oneBit = true;
        }
        i++;
    }
    return oneBit;
};

File: ruby/easy/1-bit_and_2-bit_characters.rb
# @param {Integer[]} bits
# @return {Boolean}
def is_one_bit_character(bits)
    return false if bits.length == 0
    onebit = false
    i = 0;
    while i < bits.length
        if(bits[i] == 1)
            onebit = false
            i += 1
        else
            onebit = true
        end
        i += 1
    end
    return onebit
end
